### ISSUE TITLE
R syntax in Python docs

### ISSUE THREAD

**Issue by HenrikBengtsson** [seguid/seguid-python#26]
The 'Usage' in <https://seguid-python.seguid.org/#usage> appears to have been copied as-is from R.

**Comment by BjornFJohansson** [comment]
```
  lsseguid(seq, alphabet="{DNA}", form=c("long", "short", "both"))

  csseguid(seq, alphabet="{DNA}", form=c("long", "short", "both"))

  ldseguid(watson, crick, alphabet="{DNA}", form=c("long", "short", "both"))

  cdseguid(watson, crick, alphabet="{DNA}", form=c("long", "short", "both"))

  seguid(seq, alphabet="{DNA}", form=c("long", "short", "both"))
```


Replaced  with the code below. The Python code does not spell out the allowed values for `form`. I do not know of a way to do this in this context.

```
lsseguid(seq, alphabet="{DNA}", form="long")

csseguid(seq, alphabet="{DNA}", form="long")

ldseguid(watson, crick, alphabet="{DNA}", form="long")

cdseguid(watson, crick, alphabet="{DNA}", form="long")

seguid(seq, alphabet="{DNA}", form="long")
```

**Comment by BjornFJohansson** [comment]
Would some version of this be better? These can be pasted directly into the interpreter without defining variables.

```
lsseguid(seq="ATG", alphabet="{DNA}", form="long")

csseguid(seq="ATG", alphabet="{DNA}", form="long")

ldseguid(watson="ATG", crick="CAT", alphabet="{DNA}", form="long")

cdseguid(watson="ATG", crick="CAT", alphabet="{DNA}", form="long")

seguid(seq="ATG", alphabet="{DNA}", form="long")
```

**Comment by BjornFJohansson** [comment]
The bug was fixed in 859dbe478166ae30e8d0a9f1da820caf6890bff2

**Comment by HenrikBengtsson** [comment]
LGTM
